etc/linters.json

```diff
@@ -64,7 +64,7 @@
     },
     {
         "name": "jscpd",
-        "args": "--config /etc/opt/goat/.jscpd.json *",
```

Comment:
Yep, that surely won't handle any exclusions.

Reply:
Ah good catch